haas/cli.py

```diff
@@ -89,7 +92,12 @@ def do_delete(url):
     return check_status_code(requests.delete(url))
 
 @cmd
-def serve():
+def serve(port):
+    try:
+        port = schema.And(schema.Use(int), lambda n: MIN_PORT_NUMBER <= n <= MAX_PORT_NUMBER).validate(port)
+    except:
```

Comment:
Always specify the actual error you're expecting --- in this case `schema.SchemaError`. If some error you're not expecting occurs, you want to hear about it (and not necessarily handle it the same way).